packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -262,11 +262,11 @@ export function _SignInStart(): JSX.Element {
     fields: Array<FormControlState<string>>,
   ) => {
     return signInCreatePromise.then(signInResource => {
-      if (!userSettings.saml.enabled) {
+      if (!userSettings.enterpriseSSO.enabled) {
         return signInResource;
       }
       /**
-       * For SAML enabled instances, perform sign in with password only when it is allowed for the identified user.
+       * For EnterpriseSSO enabled instances, perform sign in with password only when it is allowed for the identified user.
```

Comment:
```suggestion
       * For Enterprise SSO enabled instances, perform sign in with password only when it is allowed for the identified user.
```

Reply:
Fixed! [f0338ef](https://github.com/clerk/javascript/pull/4596/commits/f0338ef8f04ec5c3a7417f587052c136d0dcabf7)

Reply:
Also updated the function name [faa0e3c](https://github.com/clerk/javascript/pull/4596/commits/faa0e3cb76a841e4e93f17d3747ac4c6e1030fd7)